session/src/main/java/org/apache/iotdb/session/Session.java

```diff
@@ -589,6 +589,47 @@ public SessionDataSet executeRawDataQuery(List<String> paths, long startTime, lo
     }
   }
 
+  /**
+   * only: select last status from root.ln.wf01.wt01 where time >= 1621326244168; <PrefixPath> +
+   * <suffixPath> = <TimeSeries>
+   *
+   * @param suffixPath root.ln.wf01.wt01
+   * @param prefixPath status
+   * @param LastTime get the last data,whose timestamp greater than or equal LastTime
+   */
+  public SessionDataSet executeLastDataQuery(
+      List<String> suffixPath, List<String> prefixPath, long LastTime)
+      throws StatementExecutionException, IoTDBConnectionException {
+    try {
+      return defaultSessionConnection.executeLastDataQuery(suffixPath, prefixPath, LastTime);
+    } catch (RedirectException e) {
+      handleQueryRedirection(e.getEndPoint());
+      if (enableQueryRedirection) {
+        // retry
+        try {
+          return defaultSessionConnection.executeLastDataQuery(suffixPath, prefixPath, LastTime);
+        } catch (RedirectException redirectException) {
+          logger.error("redirect twice", redirectException);
+          throw new StatementExecutionException("redirect twice, please try again.");
+        }
+      } else {
+        throw new StatementExecutionException(MSG_DONOT_ENABLE_REDIRECT);
+      }
+    }
+  }
+
+  /**
+   * query eg. select last status from root.ln.wf01.wt01; <PrefixPath> + <suffixPath> = <TimeSeries>
+   *
+   * @param suffixPath query statement
+   * @param prefixPath query statement
```

Comment:
The same as above,  if we provide the params, we should explain the params correctly.